src/lib/api-client.js

```diff
@@ -86,6 +86,7 @@ export function checkOutItems(studentId, itemAddresses){
             Dispatcher.handleAction('OVERRIDE_REQUIRED');
         }else if(error === 'Invalid Admin'){
             Dispatcher.handleAction('CLEAR_ADMIN_CODE');
+            Dispatcher.handleAction('INVALID_CODE');
```

Comment:
It's not a new action, just have the same handler in the auth store subscribe to `INVALID_CODE` instead of `CLEAR_ADMIN_CODE`.